video-img.py

```diff
@@ -0,0 +1,65 @@
+import cv2
+import time
+import numpy as np
+import os
+
+
+
+def save_img(video_path, result_path, interval_num, resize_wh, rot90=False, show=False):
```

Comment:
specify the type of each variable. [https://peps.python.org/pep-0484/](https://peps.python.org/pep-0484/) for reference.